### ISSUE TITLE
Spawned in End tools & armor, and Dragon armor - are not unbreakable

### ISSUE THREAD

**Issue by SilverClock** [kwpugh/gobber_fabric#13]
The only exception to this is the End Hammer and End Excavator. No issues with those. However **everything** _is_ unbreakable if crafted normally.
Using Gobber v2.5.81 on Minecraft version 1.18.2.

**Comment by kwpugh** [comment]
Yes, they need to be crafted to be unbreakable.

**Comment by SilverClock** [comment]
Huh, ok. I can just spawn in the ingredients to craft it then. Thanks for your time!

**Comment by kwpugh** [comment]
it is meant to be played in survival.

**Comment by SilverClock** [comment]
Oh I do play on survival for the most part. I just spawned in an end tool to test out enchantments on it. The Collection enchantment from Charm doesn't seem to work on the hammer. But it works on the paxel.

I do like to spawn in items that I feel like I lost too - like due to a bug or I maybe just misplaced it. I don't want to have to craft it again when I already crafted it legitimately. I don't see a way to post this without reopening the issue! Sorry, new to using github..
